terraform/modules/aws/route53/input.tf

```diff
@@ -29,3 +29,13 @@ variable "cname_records" {
   }))
 }
 
+variable "eks" {
```

Comment:
its not about eks. Its more about alias dns  records can we create interface like
```
alias_records = {
  some_record_1 = {
     name = "some-name"
     type = "A"
     alias = {
        name = "a52257f4003464781a79a0193c4284b9-301112538.eu-west-1.elb.amazonaws.com"
        evaluate_target_health = true
     }
  }
}
```

Reply:
Fixed interface and variable names